bsp/esp32_s3_lcd_ev_board/CHANGELOG.md

```diff
@@ -32,7 +32,7 @@
     * Add new APIs for spiffs, audio in `bsp/esp32_s3_lcd_ev_board.h`
     * Add new APIs for ADC in `bsp/esp32_s3_lcd_ev_board.h`
 
-## v3.0.0 - 2023-12-02
+## v2.1.0 - 2023-12-02
```

Comment:
@Lzw655 PTAL! , there is no v3.0.0 released, it a typo?

Reply:
You are right. Thank you for correcting the error!